docs/tutorials/multinode/terraform/htcondor/resources.tf

```diff
@@ -0,0 +1,384 @@
+    scopes = ["https://www.googleapis.com/auth/cloud-platform"]
+  }
+
+  shielded_instance_config {
+    enable_integrity_monitoring = "true"
+    enable_secure_boot          = "false"
+    enable_vtpm                 = "true"
+  }
+
+  tags = ["${var.cluster_name}-submit"]
+  zone = var.zone
+}
+resource "google_compute_instance_template" "condor-compute" {
+  can_ip_forward = "false"
+
+  disk {
+    auto_delete  = "true"
+    boot         = "true"
+    device_name  = "boot"
+    disk_size_gb = "200"
+    mode         = "READ_WRITE"
+    source_image = data.google_compute_image.startimage.self_link
+    type         = "PERSISTENT"
+  }
+
+  machine_type = var.compute_instance_type
+
+  metadata = {
+    startup-script = local.compute_startup
+  }
+
+  name = "${var.cluster_name}-compute"
+
+  network_interface {
+    access_config {
+      network_tier = "PREMIUM"
+    }
+
+    network = "default"
+  }
+
+  project = var.project
+  region  = var.zone
+
+  scheduling {
+    automatic_restart   = "false"
+    on_host_maintenance = "TERMINATE"
+    preemptible         = var.use_preemptibles
+  }
+
+  service_account {
+    email = var.service_account
+    scopes = ["cloud-platform"]
+  }
+
+  tags = ["${var.cluster_name}-compute"]
+}
+resource "google_compute_instance_group_manager" "condor-compute-igm" {
+  count = var.multizone ? 0 : 1
+  base_instance_name = var.cluster_name
+  name               = var.cluster_name
+
+  project            = var.project
+  target_size        = "0"
+
+  update_policy {
+    max_surge_fixed         = 2
+    minimal_action          = "REPLACE"
+    type                    = "OPPORTUNISTIC"
+  }
+
+  version {
+    instance_template = google_compute_instance_template.condor-compute.self_link
+    name              = ""
+  }
+  timeouts {
+    create = "60m"
+    delete = "2h"
+  }
+  # Yup, didn't want to use this, but I was getting create and destroy errors. 
+  depends_on = [
+   google_compute_instance_template.condor-compute 
+  ]
+  zone = var.zone
+}
+
+resource "google_compute_region_instance_group_manager" "condor-compute-igm" {
+  count = var.multizone ? 1 : 0
+  base_instance_name = var.cluster_name
+  name               = var.cluster_name
+
+  project            = var.project
+  target_size        = "0"
+
+  update_policy {
+    max_surge_fixed         = var.numzones
+    minimal_action          = "REPLACE"
+    type                    = "OPPORTUNISTIC"
+  }
+
+  version {
+    instance_template = google_compute_instance_template.condor-compute.self_link
+    name              = ""
+  }
+  timeouts {
+    create = "60m"
+    delete = "2h"
+  }
+  # Yup, didn't want to use this, but I was getting create and destroy errors.
+  depends_on = [
+   google_compute_instance_template.condor-compute
+  ]
+  region = var.region
+}
+/*
+resource "google_compute_autoscaler" "condor-compute-as" {
```

Comment:
This should either be removed or have a leading comment explaining why we're keeping it (e.g., if this must be uncommented to enable certain behavior)